package.json

```diff
@@ -541,6 +543,11 @@
         "title": "Data Export",
         "category": "Couchbase"
       },
+      {
+        "command": "vscode-couchbase.tools.mdbExport",
```

Comment:
Shouldn't we rename this to mdbImport?

Applicable everywhere.

Reply:
Even mdbMigrate sounds better

Reply:
Changed to mdbMigrate everywhere.